strategies/attestationdata/majority/attestationdata.go

```diff
@@ -354,3 +347,7 @@ func (*Service) attestationDataLoop2(ctx context.Context,
 		Int("errored", errored).
 		Msg("Results")
 }
+
+func generateAttestationKey(attestationData *phase0.AttestationData) string {
+	return fmt.Sprintf("%s-%s-%d", attestationData.Source.Root.String(), attestationData.Target.Root.String(), attestationData.Slot)
```

Comment:
(also, sprintfing the key feels a bit wasteful, vis a vis a `struct { source, target Checkpoint }`)